load_tests/create_testing_resources/kinesis_s3_firehose/app.py

```diff
@@ -8,6 +9,7 @@
     core,
 )
 
+DESTINATION_LIST = ["", "std-"] # "" is the destination tag for logs coming from non-stdstream input
```

Comment:
I feel like if this is what it is... "DESTINATION" is the wrong word since this is an input list? Also can it not be an empty string somehow? This does not seem intuitive to understand...

Reply:
+1, `DESTINATION`sounds more like our output plugins like s3, cloudwatch..

Reply:
Can we call it DESTINATION_PREFIX_LIST? Because that is what it is. We need a separate destination for TCP and STDOUT because it is possible for error logs to get sent to stdout if the log4j tcp fails. We want to keep these error logs separate from the TCP logs, but still keep them so that we can add some validation in the future if needed. What do you think?

Reply:
I still feel that's confusing and that future maintainers of the code might not understand it- it either needs a comment explaining, or why can't it be `LOG_SOURCE_PREFIX_LIST` or `INPUT_PREFIX_LIST` or something like that? This still seems input side, not output side... can you explain how this is related to the output?

Reply:
Addressed in this new PR into tcp-load-tests https://github.com/fala-aws/aws-for-fluent-bit/pull/2

Reply:
This should help keep separate of the new, unchecked changes for the Load Test general improvements
- Extract the resource name resolution
- Enhance the markdown output
- Add bar raising logic

Reply:
Merged